purescript/src/Funkia/List/Types.js

```diff
@@ -0,0 +1,61 @@
+"use strict";
+
+var L = require("list");
+
+exports._eq = function _eq(f, xs, ys) {
+  if (L.length(xs) !== L.length(ys)) return false;
+  for (var i = 0; i < L.length(xs); i++) {
+    if (!f(L.nth(i, xs))(L.nth(i, ys))) return false;
+  }
+  return true;
+};
```

Comment:
What about adding an `equalsBy` to List itself?

Reply:
Do you see general utility for `equalsBy`?